.travis.yml

```diff
@@ -0,0 +1,11 @@
+script:
+  - if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_EVENT_TYPE" = "push" ]]; then
+      openssl aes-256-cbc -K $encrypted_a641d77e729e_key -iv $encrypted_a641d77e729e_iv
+        -in deploy/deploy_rsa.enc -out /tmp/deploy_rsa -d;
+
+      eval "$(ssh-agent -s)";
+      chmod 600 /tmp/deploy_rsa;
+      ssh-add /tmp/deploy_rsa;
+
+      ssh -o StrictHostKeyChecking=no root@$RUNNER_HOST "tsp git -C ./ruby-bench-docker pull --rebase origin master && tsp ./ruby-bench-docker/deploy/script";
```

Comment:
> The bare metal server might be running benchmarks and building can be CPU intensive affecting the results

For that reason I enqueue deploy task through `tcp`, to ensure no suite is running when deploy is in progress. Have a look at this line we are commenting on :arrow_up: